requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.12.4
```

Comment:
Does this mean we should also have a `setup` file and make this a `pip` package? Is `requirements.txt` a pip-only thing?